test/api/controllers/authController.test.js

```diff
@@ -266,4 +268,20 @@ describe('Test the auth controller', () => {
         });
     });
   });
+
+  describe('#getAuthenticationStrategies', () => {
+    it('should return a valid response', () => {
+      kuzzle.pluginsManager = new PluginManager(kuzzle);
```

Comment:
no need to do that. Kuzzle Mock already has a mocked plujginManager.
We should just add `listStrategies` method here:
https://github.com/kuzzleio/kuzzle/blob/413ff1a8e004e710df617005384a1ce945dcd1e4/test/mocks/kuzzle.mock.js#L156